server/app/com/xsn/explorer/models/persisted/BlockInfo.scala

```diff
@@ -0,0 +1,43 @@
+package com.xsn.explorer.models.persisted
+
+import com.xsn.explorer.models.values._
+import play.api.libs.json.{Json, Writes}
+
+sealed trait BlockInfo extends Product with Serializable {
```

Comment:
This could be as simple as:

```scala
  case class BlockInfo(
      hash: Blockhash,
      previousBlockhash: Option[Blockhash],
      nextBlockhash: Option[Blockhash],
      merkleRoot: Blockhash,
      height: Height,
      time: Long,
      difficulty: BigDecimal,
      transactions: Int
  )
```